src/modules/IscsiClientLib.rb

```diff
@@ -792,7 +782,15 @@ def checkInitiatorName
       ret
     end
 
-    # delete deiscovered target from database
+    # Logout from the target (ie. remove the corresponding session)
+    #
+    # This does not delete the target from nodes database. The name of the method is plain wrong
+    # for historical reasons.
```

Comment:
I would vote for just renaming it, but I suspect there's a good reason you don't do that?